src/views.py

```diff
@@ -10,11 +10,25 @@ def create(department: dict):
     return "Data added successfully"
 
 
+@router.middleware("http")
+async def print_hello(request: Request, call_next):
+    print("Hello")
+    response = await call_next(request)
+    return response
+
+
 @router.get("/", name="get_departments")
 def get_all():
     return DepartmentManager.get_all()
 
 
+@router.middleware("http")
+async def print_goodbye(request: Request, call_next):
+    print("Goodbye")
+    response = await call_next(request)
+    return response
+
+
```

Comment:
Why are you adding two middlewares? We need only one (with two prints), remove the second one please. And move it to a separate file